CHANGELOG.md

```diff
@@ -49,6 +49,7 @@ You can now add attributes to the `<body>` element of a page, by using the [`bod
 - [Pull request #1609: Update hex value for secondary text to improve contrast](https://github.com/alphagov/govuk-frontend/pull/1609).
 - [Pull request #1620: Only add underline to back link when href exists ](https://github.com/alphagov/govuk-frontend/pull/1620).
 - [Pull request #1631: Fix classes on character count when in error state](https://github.com/alphagov/govuk-frontend/pull/1631).
+- [Pull request #1638: Check component item arrays are not empty before outputting markup](https://github.com/alphagov/govuk-frontend/pull/1638).
```

Comment:
Something I missed, this changelog entry needs to be in an 'unreleased section' instead of being in the previous release

Reply:
Yeah that looks great but 'New features' should be 'Fixes'

Reply:
Sorry, this probably needs updating again as v3.4.0 has now been released.